.github/workflows/wiki-publish.yaml

```diff
@@ -20,46 +21,44 @@ jobs:
     steps:
       - uses: actions/checkout@v2
         with:
-          fetch-depth: 0  # required to calc credits list
+          fetch-depth: 0  # required to generate credits list
+
       - uses: actions/setup-python@v2
         with:
           python-version: 3.7
 
-      - name: Get Rez Version
+      - name: Build Wiki
+        working-directory: wiki
         run: |
           rezver=$(cat ./src/rez/utils/_version.py | grep -w _rez_version | tr '"' ' ' | awk '{print $NF}')
-          echo "rezver=${rezver}" >> $GITHUB_ENV
 
-      - name: Build Wiki
-        working-directory: wiki  # Start in correct dir, else inspect fails
-        run: |
           python update-wiki.py \
             --keep-temp \
             --no-push \
             --github-release=${rezver} \
             --wiki-dir="${{ github.workspace }}"/"${{ env.TEMP_WIKI_DIR }}" \
             --wiki-url="${{ env.CLONE_URL }}"
 
-      - uses: actions/upload-artifact@v1.0.0
+      - uses: actions/upload-artifact@v2
         with:
           name: wiki-markdown
           path: ${{ env.TEMP_WIKI_DIR }}
 
-
   publish:
     name: Publish to GitHub Wiki
     runs-on: ubuntu-latest
     needs: build
```

Comment:
Would it be possible to still keep the build and publish as separate jobs in the same workflow YAML file, but for the `publish` job, include an extra `if` to only run it during a `release` event?

```yaml
on:  # Merge in this at the top of file
  release:
    types: [released]


  publish:
￼    name: Publish to GitHub Wiki
￼    runs-on: ubuntu-latest
     if: github.event_name == 'release'
￼    needs: build
```